app/profiles_strings.grdp

```diff
@@ -246,7 +246,7 @@
   <message name="IDS_DEFAULT_AVATAR_LABEL_7" desc="The label for the default yellow avatar.">
     Default yellow avatar
   </message>
-  <message name="IDS_DEFAULT_AVATAR_LABEL_8" desc="The label for the the female spy avatar.">
+  <message name="IDS_DEFAULT_AVATAR_LABEL_8" desc="The label for the female spy avatar.">
```

Comment:
This file is auto-generated as well. The typo would need to be fixed in Chromium (https://source.chromium.org/chromium/chromium/src/+/main:chrome/app/profiles_strings.grdp;l=252).